pkg/volume/flexvolume/plugin.go

```diff
@@ -42,9 +42,30 @@ type flexVolumePlugin struct {
 	unsupportedCommands []string
 }
 
-var _ volume.AttachableVolumePlugin = &flexVolumePlugin{}
+type flexVolumeAttachablePlugin struct {
+	flexVolumePlugin
+}
+
+var _ volume.AttachableVolumePlugin = &flexVolumeAttachablePlugin{}
 var _ volume.PersistentVolumePlugin = &flexVolumePlugin{}
```

Comment:
These are not flex objects. These are infra objects and we are just declaring it.